content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -122,91 +122,86 @@ Note that `localhost` is required, the value itself is ignored and can be anythi
 
 The PostgreSQL connector maps the [scalar types](../components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](../components/prisma-schema/data-model) as follows to native column types:
 
+> Alternatively, see [Prisma schema reference](../../../reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
 ### Prisma Migrate
 
-| Prisma     | PostgreSQL       |
-| ---------- | ---------------- |
-| `String`   | `text`           |
-| `Boolean`  | `boolean`        |
-| `Int`      | `integer`        |
-| `Float`    | `decimal(65,30)` |
-| `Datetime` | `timestamp`      |
-| `Json`     | `jsonb`          |
+| Prisma     | PostgreSQL         |
+| ---------- | ------------------ |
+| `String`   | `text`             |
+| `Boolean`  | `boolean`          |
+| `Int`      | `integer`          |
+| `BigInt`   | `bigint`           |
+| `Float`    | `double precision` |
+| `Decimal`  | `decimal(65,30)`   |
+| `Datetime` | `timestamp(3)`     |
+| `Json`     | `jsonb`            |
+| `Bytes`    | `bytea`            |
 
 ### Introspection
```

Comment:
This is now the main place where users can find all the native type mappings for a given connection (previously in `/concepts/components/preview-features/native-types/native-types-mappings`) 

I's suggest changing the title of this section from "Introspection" to "Native type mappings" (here and in the MySQL connector page) so that it's more easily found.

Reply:
Great idea - renaming headings now